olla/include/openmm/Platform.h

```diff
@@ -45,23 +45,42 @@ class Kernel;
 class KernelFactory;
 
 /**
- * A Platform defines an implementation of all the kernels needed to perform some calculation.
- * More precisely, a Platform object acts as a registry for a set of KernelFactory
- * objects which together implement the kernels.  The Platform class, in turn, provides a
- * static registry of all available Platform objects.
+ * A Platform defines an implementation of all the kernels needed to perform
+ * some calculation. More precisely, a Platform object acts as a registry for a
+ * set of KernelFactory objects which together implement the kernels.  The
+ * Platform class, in turn, provides a static registry of all available
+ * Platform objects.
  *
- * To get a Platform object, call
+ * OpenMM ships with platforms that provide high-performance simulation on a CPU
+ * or GPU. They are described in more detail in the User Manual. Usually, the
+ * platform is chosen automatically when a Context is constructed.
  *
+ * There are three main ways to get a Platform object:
+ *
+ * 1. If the name of the Platform is known, call getPlatformByName():
+ * \verbatim embed:rst:leading-asterisk
+ *     .. code-block:: cpp
+ *
+ *        Platform& platform = Platform::getPlatformByName(name);
+ * \endverbatim
+ *
+ * 2. If a Context has already been constructed, call Context::getPlatform():
  * \verbatim embed:rst:leading-asterisk
- * .. code-block:: cpp
+ *     .. code-block:: cpp
+ *
+ *        Platform& platform = context.getPlatform();
+ * \endverbatim
  *
- *    Platform& platform = Platform::findPlatform(kernelNames);
+ * 3. If the required kernels are known, call findPlatform():
```

Comment:
In practice there is almost never a reason for someone to call this method.  It exists for use by ContextImpl, to automatically select a Platform if the user didn't specify one.  It's still ok to document it, just mention that people are unlikely to need it.  (Although it probably isn't accurate to describe it as one of the "main ways" to get a Platform.)

Reply:
Prior to these changes, this was the only method documented. I think what might have happened is that Platform was originally documented for internal use, and was later made part of the public API and the documentation was never updated. I'll remove it in the upcoming PR